message_definitions/v1.0/development.xml

```diff
@@ -672,6 +691,16 @@
       <field type="float" name="temperature" units="K" invalid="NaN">Fuel temperature. NaN: field not provided.</field>
       <field type="uint32_t" name="fuel_type" enum="MAV_FUEL_TYPE">Fuel type. Defines units for fuel capacity and consumption fields above.</field>
     </message>
+    <message id="398" name="GENERIC_PAYLOAD_STATUS">
+      <description>Generic payload status.</description>
+      <field type="uint32_t" name="uptime_ms" units="ms">Time since the start-up of the generic payload in ms</field>
+      <field type="uint8_t" name="enable">0: Generic payload OFF, 1: Generic payload ON</field>
```

Comment:
we either need this message to have an index field to correspon to the FUNCTIONS_CONTROL message, or if this is communicating the payload power on/off then we need to have another separate message to publish the status of the FUNCTIONS_CONTROL state

Reply:
@thomas-watters-skydio Uploaded changes to include a new function status message. Let me know what you think! :)